pkg/pillar/hypervisor/hypervisor.go

```diff
@@ -65,6 +65,13 @@ func GetHypervisor(hint string) (Hypervisor, error) {
 	}
 }
 
+// Check if the path exists.
+func isenabled(path string) bool {
```

Comment:
I think we already have a function for this in some package in pillar. Probably called somethingExists()

Reply:
Let me check.

Reply:
pillar/utils/file has FileExists() will use that